webpages/settings/index.html

```diff
@@ -307,24 +329,34 @@ <h1>{{ msg("settings") }}</h1>
                     :data-setting-id="setting.id"
                     spellcheck="false"
                   />
-                  <picker
-                    v-if="setting.type === 'color'"
-                    :value="addonSettings[addon._addonId][setting.id] || setting.default"
-                    :setting="setting"
-                    :addon="addon"
-                    @click=""
-                    :no_alpha="!setting.allowTransparency"
-                    v-click-outside="closePickers"
-                  ></picker>
-                  <button
-                    type="clear-button"
-                    class="large-button clear-button"
-                    :disabled="!addon._enabled"
-                    v-if="setting.type === 'color'"
-                    @click="updateOption(setting.id, setting.default, addon)"
-                  >
-                    {{ msg("reset") }}
-                  </button>
+                  <template v-if="setting.type === 'color'">
```

Comment:
I'm confused on why there's a template element inside the body

Reply:
https://v3.vuejs.org/guide/conditional.html#conditional-groups-with-v-if-on-template